### ISSUE TITLE
Any help is needed?

### ISSUE THREAD

**Issue by babysor** [lucidrains/soundstorm-pytorch#10]
I found this repo is quite active and you may need someone to do parallel experiment or coding. I'm willing to help and working on getting context of the whole repo.

**Comment by lucidrains** [comment]
@babysor yea, just jump in with a pull request

@Diaz1980 no there is not

**Comment by Diaz1980** [comment]
> @babysor yea, just jump in with a pull request
> 
> @Diaz1980 no there is not

Then how can we test the inference of the SoundStorm model, is there are any google colab notebook or hugging face spaces to see  the results

**Comment by therealadityashankar** [comment]
@Diaz1980 the project is a code implementation afaiui, it hasn't been trained to the google-degree yet, and that's going to cost alot of money over time, its for if someone else wants to train soundstorm

**Comment by anacondabitch** [comment]
yes i have an issue.. what are you talking about in [README.md] ???
 do you mind sharing  a proper colab Jupyter file so we can run in cloud and test it out thanks
also from what i understand this is complete pipeline for TTS and not just vocoder or something right?

**Comment by chunlim08** [comment]
I am sorry to say that I am confused with what displayed in [README.md]...
Would you mind to share more about the code and how it works (including data input and outcomes)
I would appreciate it if you could share a proper colab Jupyter file or notebook as well as more details so that we can test it out
Much appreciate it!

**Comment by Curlypla** [comment]
It's an implementation that isn't finished yet, and when it is (which can take a long time), someone has to train it, which costs a lot of money. So there's no colab and no code to get results, because the implementation isn't finished and so there's no model.
